mysql-test/r/all_persisted_variables.result

```diff
@@ -52,9 +52,9 @@ include/assert.inc ['Expect 413 persisted variables in the table. Due to open Bu
 ************************************************************
 # restart
 
-include/assert.inc ['Expect 353 persisted variables in persisted_variables table.']
-include/assert.inc ['Expect 353 persisted variables shown as PERSISTED in variables_info table.']
-include/assert.inc ['Expect 353 persisted variables with matching peristed and global values.']
+include/assert.inc ['Expect 410 persisted variables in persisted_variables table.']
```

Comment:
60 variable difference?!

Reply:
Here, assert **message** hasn't been updated for a long time. For some reason, upon adding a new var, developers were updating only assert **condition**. Before this commit the message should have been set to 409, but it wasn't.